doc/devel/wmgr-api.rst

```diff
@@ -76,8 +76,7 @@ creating/deleting databases and tables, loading table data, etc.
             * 500 - for database errors
 
         Response body (for successful completion):
-            JSON object, "results" property is a list of database
-            objects with keys:
+            JSON object, "results" property is a list of database objects with keys:
```

Comment:
I can't comment on the individual lines in this PR, but be aware that this entire file is misformatted. It shouldn't be indented because semantically that indicates levels of quotation. Unindent all text except for the content of "Response headers", "Status Codes", "Response body", which would then become definition list items.

Reply:
I did not write this page, so I do not want to change it in order not to introduce errors. @andy-slac could you please have a look at @jonathansick remark and take it in account in a new ticket?